### DISCUSSION TITLE
Hide welcome message

### DISCUSSION THREAD

**Discussion by blueblazer22x** [croneter/PlexKodiConnect#1787] [Q&A]
Is it possible to turn off this welcome message?  I can't find an option to do so.

I've turned off "Show syncing progress" but that doesn't hide the welcome message.

![20220201_112614](https://user-images.githubusercontent.com/81867386/152014957-5a9d35fc-b9d3-4e43-b6d4-d2f9f38d0caf.jpg)
?

**Comment by hamakavula** [comment]
Indeed this option doesn't work. Same thing goes for the Option "Display message if PMS goes offline".
Kodi 19.3 + PKC 3.6.14

**Comment by JCalvi** [comment]
I agree and added an issue for it here https://github.com/croneter/PlexKodiConnect/issues/1916
You can make the code edit to service_entry.py if you want to yourselves for now.
JC.

**Comment by JCalvi** [comment]
I tried this to get rid of the screen flicker when PKC loads.
It is definitely PKC, it happens about 1-2sec after the welcome banner appears, and again about 2 sec after it leaves.
Happens on two separate systems with different displays and delaying PKC startup moves the flicker by the delay. 
Disabling PKC gets rid of it.
Disabling the banner does not change it though, so it is not the actual banner.
I have tried commenting out various skin refresh lines in PKC but still haven't found the actual cause yet.
It's probably related to sync at startup because the first flicker is at the start of sync and second at the end.
Displaying the sync info or not has no effect on it.
